api/endpoints/question.py

```diff
@@ -65,5 +67,23 @@ def question_sidewalk_search():
             return jsonify({ "results": permits })
         return jsonify({ "results": [] })
 
+    @app.route("/question/ridetrips")
```

Comment:
Similarly, can we rename this endpoint to something like `/question/ohare/rideshare`?